lib/aurora_bootstrapper/notifier.rb

```diff
@@ -12,7 +14,34 @@ def export_date
     
     # ENV is string to string dictionary
     def export_date_override
-      DateTime.now.strftime("%Y-%m-%d") if ENV.key?('EXPORT_DATE_OVERRIDE')
+      return nil unless ENV.key?('EXPORT_DATE_OVERRIDE')
+
+      today = Date.today
+      days_ago = (0..30).find do | days_ago |
+        exists_export? date: today-days_ago
+      end
+      
+      if days_ago.nil? || days_ago == 0
+        today.strftime( DATE_FORMAT )
+      else
+        ( today - days_ago.to_i + 1).strftime( DATE_FORMAT )
+      end
+    end
+
+    def exists_export?( date: )
+      prefix = [ bucket_path, date.strftime( DATE_FORMAT ) ].join( '/' )
+
+      found_object = client.list_objects_v2({ 
+        bucket: bucket,
+        prefix: prefix }).contents.find do | object | 
+          object.key.include? "DONE"
+        end
+
+      if found_object.nil?
+        AuroraBootstrapper.logger.info( message: "No objects in bucket '#{bucket}/#{prefix}'." )
```

Comment:
can you tell me a little about how we use this message?

Reply:
Originally I thought it could indicate some potential error ? 
But I can remove it for now.